modules/storage-rocksdb/src/main/java/org/apache/ignite/internal/storage/rocksdb/RocksDbMvPartitionStorage.java

```diff
@@ -137,6 +140,15 @@ public class RocksDbMvPartitionStorage implements MvPartitionStorage {
     /** Maximum size of the key. */
     private static final int MAX_KEY_SIZE = ROW_PREFIX_SIZE + HYBRID_TIMESTAMP_SIZE;
 
+    /** Garbage collector's queue key's timestamp offset. */
+    private static final int GC_KEY_TS_OFFSET = Short.BYTES;
+
+    /** Garbage collector's queue key's row id offset. */
```

Comment:
There is no point to do it for constants because they can change (while documentation can become stale) and IntelliJ IDEA calculates constant's value
<img width="694" alt="image" src="https://user-images.githubusercontent.com/4058545/216582307-8c8e4249-0542-41fa-a843-46c5b81a74da.png">

Reply:
Ok